cmd/promtool/main.go

```diff
@@ -340,8 +340,8 @@ func main() {
 	}
 
 	for _, f := range *featureList {
-		opts := strings.Split(f, ",")
-		for _, o := range opts {
+		opts := strings.SplitSeq(f, ",")
+		for o := range opts {
```

Comment:
nit: guessing a quirk of the tool but sometimes there's a temporary and sometimes not...